From ac5b38bdb3c870afa5c4999486afc6c32157a93b Mon Sep 17 00:00:00 2001
From: Richard Liu <r66033@freescale.com>
Date: Thu, 27 Sep 2012 14:16:52 +0800
Subject: [PATCH 066/281]  ENGR00175943 disable NEON memcpy on imx6

disable NEON memcpy on imx6 as neon memcpy performance is worse than non-neon

Signed-off-by: Richard Liu <r66033@freescale.com>
---
 imx6/BoardConfigCommon.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/imx6/BoardConfigCommon.mk b/imx6/BoardConfigCommon.mk
index 29d0ff2..aba41c9 100644
--- a/imx6/BoardConfigCommon.mk
+++ b/imx6/BoardConfigCommon.mk
@@ -46,6 +46,9 @@ PREBUILT_FSL_IMX_CODEC := true
 PREBUILT_FSL_IMX_OMX := false
 PREBUILT_FSL_IMX_GPU := true
 
+# use non-neon memory copy on mx6x to get better performance
+ARCH_ARM_USE_NON_NEON_MEMCPY := true
+
 # for kernel/user space split
 # comment out for 1g/3g space split
 # TARGET_KERNEL_2G := true
-- 
1.8.0

